lighthouse-core/computed/metrics/tbt-utils.js

```diff
@@ -0,0 +1,57 @@
+/**
```

Comment:
@patrickhulce ? keeping it near where its used seem good to me

Reply:
I'm also +1 to keeping it near the metrics, since it's effectively the metric implementation just de-cycled

Reply:
worth splitting into its own PR? Self contained and it would give an opportunity to split the tests in `total-blocking-time-test.js` that are just of the utils into `tbt-utils-test.js`